spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/pulsar/PulsarProperties.java

```diff
@@ -250,13 +250,43 @@ public Authentication getAuthentication() {
 
 	public static class Defaults {
 
+		/**
+		 * Default tenant to use when producing or consuming messages against a
+		 * non-fully-qualified topic URL. When not specified Pulsar uses a default tenant
+		 * of 'public'.
+		 */
+		private String tenant;
+
+		/**
+		 * Default namespace to use when producing or consuming messages against a
+		 * non-fully-qualified topic URL. When not specified Pulsar uses a default
+		 * namespace of 'default'.
+		 */
+		private String namespace;
+
```

Comment:
As these properties both relate to topics, it would be good to put them both in a `topic` group so they'd become `spring.pulsar.defaults.topic.tenant` and `spring.pulsar.defaults.topic.namespace`. I think that still makes sense given that `typeMappings` is partially topic related. What do you think though, @cbono?

Reply:
Great suggestion as this also gives way to adding an `enabled` flag to the `spring.pulsar.defaults.topic` to disable default topics if so desired.